app/admin/quests/page.tsx

```diff
@@ -91,7 +92,9 @@ export default function Page() {
           </div>
         </div>
         {loading ? (
-          <FeaturedQuestSkeleton />
+          <Loading isLoading={loading} loadingType="skeleton">
+            <FeaturedQuestSkeleton />
+          </Loading>
```

Comment:
What's the point of creating <Loading /> to do that ? 

Here you need to wrap the section directly and delete the ternary with the loading like so:
```tsx
          <Loading isLoading={loading} loadingType="skeleton">
            <section className={homePagestyles.section}>
            ....
```